ast.go

```diff
@@ -14,6 +14,14 @@ func astAssign(left, right ast.Expr) ast.Stmt {
 	}
 }
 
+func astAssignMulti(left, right []ast.Expr) ast.Stmt {
```

Comment:
Variation of `astAssign` that works for generating `foo, bar := gir, zim` instead of just `foo := gir`